cli/internal/auth/login_okta.go

```diff
@@ -0,0 +1,191 @@
+package auth
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/base64"
+	"fmt"
+	"math/rand"
+	"net"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/cli/browser"
+	"github.com/llmariner/llmariner/cli/internal/accesstoken"
+	"github.com/llmariner/llmariner/cli/internal/configs"
+	llmcontext "github.com/llmariner/llmariner/cli/internal/context"
+	"golang.org/x/oauth2"
+)
+
+func oktaLogin(ctx context.Context, c *configs.C, noOpen bool) error {
+	port := int(8084)
+	listerner, err := net.Listen("tcp", fmt.Sprintf(":%d", port))
+	if err != nil {
+		return fmt.Errorf("listen: %v", err)
+	}
+
+	cli, err := newOktaClient(ctx, c.Auth)
+	if err != nil {
+		return fmt.Errorf("new okta client: %v", err)
```

Comment:
```suggestion
		return fmt.Errorf("new okta client: %s", err)
```

This is copied from the existing code, but how about using `%s`?